frontend/src/services/sessionStorageService.ts

```diff
@@ -0,0 +1,20 @@
+export const saveUserToStorage = (user: {}) => {
```

Comment:
still a question - `export const saveUserToStorage = (user: {}) => {` why we using this type?